src/components/CalendarMonthGrid.jsx

```diff
@@ -32,6 +33,7 @@ const propTypes = {
   onDayTouchTap: PropTypes.func,
   onMonthTransitionEnd: PropTypes.func,
   transformValue: PropTypes.string,
+  daySize: PropTypes.number,
```

Comment:
should this perhaps be a finite, positive integer? If so, `airbnb-prop-types` has `nonNegativeInteger` for just this use case :-D